app/repository/geography.py

```diff
@@ -61,18 +60,17 @@ def get_geo_subquery(
 
 
 def count_families_per_category_in_each_geo(
-    db: Session, allowed_corpora: list[str]
+    db: Session,
+    allowed_corpora: Optional[list[str]],
```

Comment:
`Optional[list[str]]` -> does "empty list" (`[]`) suffice to say "don't filter"? From the comment, it suggests so.

If that is the case, could we simplify this by just having `list[str]`?

If not - what is the difference between `[]` and `None`?